packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -710,24 +769,79 @@ const runGenerator = (context, {outputDir, templateVersion, verbose}) => {
         const assetsDir = p.join(ASSETS_TEMPLATES_DIR, id)
         if (sh.test('-e', assetsDir)) {
             getFiles(assetsDir)
-                .map((file) => file.replace(assetsDir, ''))
-                .forEach((relFilePath) =>
+                .map((file) => {
+                    const relFilePath = file.replace(assetsDir, '')
+                    return relFilePath
+                })
+                .forEach((relFilePath) => {
                     processTemplate(relFilePath, assetsDir, outputDir, context)
+                })
+        }
+
+        let extensionSamplePath
+        if (extractExtension) {
+            const extensionTmp = fs.mkdtempSync(p.resolve(os.tmpdir(), 'extract-extension-sample'))
+            const extensionTarFile = sh
+                .exec(
+                    `npm pack @salesforce/extension-sample@1.0.0-dev --pack-destination="${extensionTmp}"`,
```

Comment:
I suspect that we shouldn't have explicit references to concrete extensions here. Ideally when the question of what application extensions you want to use is asked, the answers answered would be a comma separated list of package names for those Application Extensions.